code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -940,6 +940,43 @@
 			// +20% success propability on each step, useful while operating in less-than-perfect conditions
 	..()
 
+/datum/reagent/space_cleaner/bleach
+	name = "Bleach"
+	description = "A harsh chemical sterilizer. Do Not Ingest."
+	color = "#fbfdd6"
+	taste_description = "burning"
+	toxpwr = 0 //all the damage comes from the life process
+
+/datum/reagent/space_cleaner/bleach/reaction_mob(mob/living/carbon/C, method=TOUCH, reac_volume)
+	. = ..()
+	if(method in list(TOUCH, VAPOR))
+		for(var/s in C.surgeries)
+			var/datum/surgery/S = s
+			S.success_multiplier = max(0.1, S.success_multiplier)
+			// +10% success propability on each step, useful while operating in less-than-perfect conditions
+	..()
+
+/datum/reagent/space_cleaner/bleach/on_mob_life(mob/living/carbon/M) //stronger cyanide, basically
```

Comment:
it's stronger cyanide but it's easier to make. Doesn't make much sense

Reply:
unfortunately the chems I *would* need for it to be harder and more realistic, are actually used by similar reactions or not in the code.

Reply:
also chilling outside of chem is difficult to pull off